packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx

```diff
@@ -174,52 +174,54 @@ export const NavDrawerDefault = (props: Partial<NavDrawerProps>) => {
             Interviews
           </NavItem>
           <Caption1Strong className={styles.headingContent}>Benefits</Caption1Strong>
-          <NavItem icon={<HealthPlans />} value="10">
+          <NavItem href="https://www.bing.com" onClick={someClickHandler} icon={<HealthPlans />} value="10">
             Health Plans
           </NavItem>
           <NavCategory value="11">
             <NavCategoryItem icon={<Person />} value="12">
               Retirement
             </NavCategoryItem>
             <NavSubItemGroup>
-              <NavSubItem target="_blank" onClick={someClickHandler} value="13">
+              <NavSubItem href="https://www.bing.com" value="13">
                 Plan Information
               </NavSubItem>
-              <NavSubItem target="_blank" onClick={someClickHandler} value="14">
+              <NavSubItem href="https://www.bing.com" value="14">
                 Fund Performance
               </NavSubItem>
             </NavSubItemGroup>
           </NavCategory>
 
           <Caption1Strong className={styles.headingContent}>Learning</Caption1Strong>
-          <NavItem icon={<TrainingPrograms />} value="15">
+          {/* As button, since no href is present */}
+          <NavItem icon={<TrainingPrograms />} as="button" onClick={someClickHandler} value="15">
             Training Programs
           </NavItem>
           <NavCategory value="16">
             <NavCategoryItem icon={<CareerDevelopment />}>Career Development</NavCategoryItem>
             <NavSubItemGroup>
-              <NavSubItem target="_blank" onClick={someClickHandler} value="17">
+              <NavSubItem href="https://www.bing.com" onClick={someClickHandler} value="17">
                 Career Paths
               </NavSubItem>
-              <NavSubItem target="_blank" onClick={someClickHandler} value="18">
+              {/* As button, since no href is present */}
+              <NavSubItem as="a" onClick={someClickHandler} value="18">
```

Comment:
shouldn't this be
```suggestion
              <NavSubItem onClick={someClickHandler} value="18">
```

Reply:
yeah, these are here as test cases. Can remove.